### DISCUSSION TITLE
夸克网盘

### DISCUSSION THREAD

**Discussion by zcb174** [AlistGo/alist#4823] [General]
夸克网盘存在不定时的掉线情况
通常一部两小时的电影，在线看完都会掉线
大概显示为alist挂载的夸克网盘一直转圈
此时其他网盘正常
重启alist夸克恢复正常使用
这个情况，大家遇到过吗，怎么解决

**Comment by Chenjun-Shen** [comment]
没用过夸克网盘。出现问题的时候，alist的服务器端有什么output日志吗？

**Comment by SeanHeuc** [comment]
因为原来代码没有完整、标准实现http method header 等，某些请求会出问题。 这个已经修好了，但是合并到主分支要等等，你方便测试一下新功能吗，帮忙测试可以加快进度

**Reply by Chenjun-Shen** [reply]
不好意思，我没有夸克网盘的账号。如果有日志提供，可以帮忙给点参考建议。
